java/simulator/src/main/java/com/hazelcast/simulator/agent/workerprocess/WorkerParameters.java

```diff
@@ -52,6 +52,31 @@ public String findDriver() {
         }
     }
 
+    public String findDriverClass() {
+        String driverClassOverride = findDriverClassOverride();
+        String driver = findDriver();
+        return driverClassOverride != null ? driverClassOverride : switch (driver) {
+            case "hazelcast-enterprise3" -> "com.hazelcast.simulator.hazelcast3.Hazelcast3Driver";
```

Comment:
I know it's being used like that previously, but probably we can improve that by extracting to an Enum?